### PR TITLE
Playbooks x Quest Chains integration

### PR DESCRIPTION
**Pull request by vidvidvid**

### CODE REVIEW

**Comment by alalonde** [comment]
Looks like there are some build errors here @vidvidvid

**Comment by vidvidvid** [comment]
@alalonde yeah, sorry this should be a draft for now

**Comment by vidvidvid** [comment]
The PR is ready. There is, however, a bug which is messing up the css of the MarkdownEditor and looks like this: 
Screenshot 2023-04-03 at 23 02 38

It looks like there might be a problem in importing the css files in `web/pages/_app.tsx`:
```
import 'react-markdown-editor-lite/lib/index.css';
import 'assets/custom-markdown-editor.scss';
```
The classes from these two files aren't showing up in the console, so I'm assuming it has to do something with this. I don't know how to solve this problem, nor how it got messed up in the first place. On master branch everything works fine.
Help, please

File: packages/web/utils/questChains.ts [hunk 1]
```diff
@@ -193,6 +231,144 @@ export const QuestChainGreatHousesDetails: Record<
   },
 };
 
+export const QuestChainPlaybooksDetails: Record<
+  QuestChainType,
+  QuestChainInfo
+> = {
+  [QuestChainsPlaybooks.MEME_DRIVEN_ORGANIZATIONS]: {
```
Comment (alalonde) [inline packages/web/utils/questChains.ts:238]: Seems like these playbooks should be in a database table... Do you have a good reason why they shouldn't be?  I assume we will want to be modfiying them fairly often, which shouldn't require mucking in the code

Comment (vidvidvid) [reply packages/web/utils/questChains.ts:238]: I can't set up the hasura console on my mac, been at it for hours and then I gave up. Also it's convenient and simpler to change the config if it's in the client.

Comment (alalonde) [reply packages/web/utils/questChains.ts:238]: OK. Definitely don't agree with this 

> Also it's convenient and simpler to change the config if it's in the client.

when you can just update the data in Hasura and have it reflect in any environment in a matter of seconds...

Comment (alalonde) [reply packages/web/utils/questChains.ts:238]: @dysbulic Any thoughts here? Should we get this stuff in the DB or merge as-is

Comment (vidvidvid) [reply packages/web/utils/questChains.ts:238]: > when you can just update the data in Hasura and have it reflect in any environment in a matter of seconds...

don't you have to write a migration?

Comment (vidvidvid) [reply packages/web/utils/questChains.ts:238]: Anyways, I can't even spin up a hasura console.

**Pull request merged**